debezium-quarkus-outbox-common/pom.xml

```diff
@@ -14,6 +14,14 @@
     <name>Debezium Quarkus :: Outbox :: Common</name>
     <packaging>pom</packaging>
 
+    <properties>
+        <!-- Java version used for Debezium specific components - Server, Operator, Outbox -->
+        <debezium.java.specific.target>21</debezium.java.specific.target>
```

Comment:
@Naros This should not be overridden here and in other places. The plan is that as soon as we have Java 17 settled for everything then this property will be changed centrally via https://github.com/debezium/debezium/pull/5658